src/nft/components/details/detailsV2/DataPageTable.tsx

```diff
@@ -3,7 +3,9 @@ import { Trans } from '@lingui/macro'
 import { ActivityTableContent } from './ActivityTableContent'
 import { ListingsTableContent } from './ListingsTableContent'
 import { OffersTableContent } from './OffersTableContent'
-import { Tab, TabbedComponent } from './TabbedComponent'
+import { Tab, TabbedComponent, TabTitleWithBubble } from './TabbedComponent'
+
+export const TableContentHeight = 568
```

Comment:
I don't think this should be in this file

Reply:
Named the height to reduce magic number use and for consistency across the tabs. This seems like the best central source for it imo.

Reply:
yeah I understand I just think that if its being used in the same context of `<div style={{ height: TableContentHeight }} />` then it just might be better off to make a styled component with that set height and import the styled component